src/pages/EmployeePage/components/Languages/Languages.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+type LanguagesProps = {
+  languagesData: Language[]
+}
+
+const Languages: React.FC<LanguagesProps> = ({ languagesData }) => {
+  const langs = languagesData.map((l) => <div key={l.id}>{l.language_name}</div>)
```

Comment:
letter l - so meaningful

Reply:
I will fix it as soon as I start developing this part of functionality, but I got your point